appcenter-push/Push.d.ts

```diff
@@ -1,6 +1,6 @@
 export interface PushNotification {
-    message?: string | null;
-    title?: string | null;
+    message: string | null;
+    title: string | null;
     customProperties?: { [name: string]: string };
```

Comment:
is the question mark meaning undefined? If yes then customProperties is actually always defined and empty map if no custom property.

Reply:
Question mark means optional, in effect it is undefined when not set. 
Technically the better thing here would've been: `string | null | undefined` since it's never created by a library consumer. 

Anyway, I'll remove the question from customProperties, cheers!